sylvester/sylvester.d.ts

```diff
@@ -0,0 +1,825 @@
+﻿// Type definitions for sylvester 0.1.3
+// Project: https://github.com/jcoglan/sylvester
+// Definitions by: Stephane Alie <https://github.com/StephaneAlie>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+// === Sylvester ===
+// Vector and Matrix mathematics modules for JavaScript
+// Copyright (c) 2007 James Coglan
+
+interface VectorStatic {
```

Comment:
While doing the def for proj4 I realized we could hide those interface in the module, otherwise they show up in the global scope, can you give it a try by moving all interfaces inside the Sylvester module? I'm guessing the global function might have to prefix some typing with Sylvester but it should work.